news/2021-05-25-moving-to-libera.md

```diff
@@ -0,0 +1,25 @@
+---
+title: HackSoc's IRC Channels Are Moving To libera.chat
+---
+We're moving!
+
+Due to recent events at Freenode, as summarised [here][summary], 
+HackSoc will be moving all its irc channels from Freenode to 
+[Libera Chat][libera].
+
+The #hacksoc channel on Libera Chat will be bridged to slack in much 
+the same way as the current Freenode channel is. 
+
+#cs-york also has a channel on Libera Chat, with the same people as 
+usually hang out there. 
```

Comment:
This should be either "with the same people as usual hanging out there" or "where the same people as usual hang out", no?

Reply:
I do not think that going back and forth over tiny bits of phrasing is helpful at this juncture